_config.yml

```diff
@@ -1,13 +1,38 @@
+address: 
+  - Address Line 1
+  - Address Line 2
+  - City, State, AZ 12345
```

Comment:
It's a little confusing going between the branding site and the stuff your
team puts out, because one appears to be lagging the other.  Is there an
up-to-date reference to show me where the standard is moving to?

Happy to pull that code out if you think it makes sense.

Thanks,

On Tue, Mar 15, 2016 at 9:53 AM, Ryan Raub <notifications@github.com> wrote:

> In _config.yml
> <https://github.com/gios-asu/ASU-Web-Standards-Jekyll-Theme/pull/6#discussion_r56198578>
> :
>
> > @@ -1,13 +1,38 @@
> > +address:
> > +  - Address Line 1
> > +  - Address Line 2
> > +  - City, State, AZ 12345
>
> The address in the footer is actually getting removed from the standards,
> it hasn't been formally announced yet but its forthcoming. The reason for
> this is it has confused Google in the past as to where 'ASU' is. eg: the
> honors college <https://barretthonors.asu.edu/>
>
> Personally I'm a little conflicted with this since our address in the
> footer gets quite a bit of use internally.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
>
> https://github.com/gios-asu/ASU-Web-Standards-Jekyll-Theme/pull/6/files#r56198578
>